simulator/docs/external-scheduler.md

```diff
@@ -0,0 +1,82 @@
+## the external scheduler
+
+This document describes how to use the external scheduler instead of the scheduler running in the simulator.
+We use the [`externalscheduler` package](../pkg/externalscheduler).
+
```

Comment:
It is helpful for the user to specify at the beginning "what the user will be able to do" with this setting to help them decide whether to use it or not.